pkg/sdn/plugin/registry.go

```diff
@@ -263,6 +263,34 @@ func (registry *Registry) GetEgressNetworkPolicies() ([]osapi.EgressNetworkPolic
 	return policyList.Items, nil
 }
 
+// Returns true to indicate status update is needed
+type UpdateNodeFunc func(node *kapi.Node) bool
```

Comment:
I don't think this layer is helpful… I would just do the Get/RetryOnConflict in the single caller